mcodingbot/plugins/peps.py

```diff
@@ -1,17 +1,26 @@
+from __future__ import annotations
+
 import re
+from contextlib import suppress
+from datetime import datetime, timedelta, timezone
 
 import crescent
 import hikari
+from cachetools import TTLCache
 from crescent.ext import tasks
 
 from mcodingbot.config import CONFIG
 from mcodingbot.utils import PEPManager, Plugin
 
-plugin = Plugin()
-pep_manager = PEPManager()
-
 PEP_REGEX = re.compile(r"pep[\s-]*(?P<pep>\d{1,4}\b)", re.IGNORECASE)
 DISMISS_BUTTON_ID = "dismiss"
+MAX_AGE_FOR_SEND = timedelta(minutes=1)
```

Comment:
I think 2 minutes is a better choice here. I doubt that cache will get that full so memory shouldn't be a problem + editing a message after 1 minute is not that crazy if the channel isn't really moving.
```suggestion
MAX_AGE_FOR_SEND = timedelta(minutes=2)
```